lib/cesium_link_web/live/link_live/index.html.heex

```diff
@@ -15,30 +15,55 @@
 </.header>
 
 <.table id="links" rows={@streams.links} phx-hook="Sorting">
-  <:col :let={{_id, _link}}><.icon name="hero-bars-3 cursor-pointer ml-4" class="handle w-5 h-5" /></:col>
+  <:col :let={{_id, link}}>
+    <.icon name="hero-bars-3 cursor-pointer ml-4" class={"handle w-5 h-5 #{if publish_in_future?(link), do: "opacity-50"}"} />
+  </:col>
+
   <:col :let={{_id, link}} label="Name">
-    <p class="font-semibold text-zinc-900"><%= link.name %></p>
+    <p class={"font-semibold text-zinc-900 #{if publish_in_future?(link), do: "opacity-50"}"}><%= link.name %></p>
```

Comment:
Avoid repeating `publish_in_future`  in every `p` pre-calculate this before when you fetch and assign links

```suggestion
    <p class={"font-semibold text-zinc-900 #{if publish_in_future?(link), do: "opacity-50"}"}><%= link.name %></p>
```